myhpom/templates/myhpom/dashboard.html

```diff
@@ -6,88 +6,111 @@
+                </p>
 
-            <div class="bg-light dashboard-profile-information__organ-donor-status-box">
-                <div class="big-asterisk">
-                    *
-                </div>
-                <div class="div">
-                    <div class="dashboard-profile-information__organ-donor-status-info">
-                        <span class="h5">I am not an organ donor.</span>
-                        <a class="small" href="/">Edit status</a>
+                <div class="dashboard-profile-information__organ-donor-status-box">
+                    <div>
+                        <img class="dashboard-profile-information__organ-icon" src="{% static 'myhpom/img/donor/icon-donor@3x.png' %}" alt="">
                     </div>
                     <div>
-                        <a href="/" class="small">
-                            Learn how you can be a donor.
-                        </a>
+                        <div class="dashboard-profile-information__organ-donor-status-info">
+                            <span class="h5 dashboard-profile-information__status">I am not an organ donor.</span>
+                        </div>
+                        <div>
+                            <a href="/" class="dashboard-profile-information__organ-donor-link">
+                                Learn how you can be a donor.
+                            </a>
+                        </div>
+                    </div>
+                    <div class="h6 dashboard-profile-information__organ-donor-edit-link">
+                        <a href="/" class="small"><span class="oi oi-pencil"></span> Edit</a>
                     </div>
                 </div>
             </div>
         </section>
 
         <hr/>
 
-        <section class="dashboard__section dashboard-health-network">
+        <section class="mb-5 dashboard__section dashboard-health-network">
             <div class="h6 float-right">
-                <a href="/" class="small">Update My Health Network</a>
+                <a href="/" class="small"><span class="oi oi-pencil"></span> Edit</a>
             </div>
 
             <h2 class="h6 dashboard__section-title">
                 Health network
             </h2>
 
-            <div class="bg-light dashboard-health-network__info-box">
-                <div class="h3">{% firstof request.user.userdetails.health_network.name request.user.userdetails.custom_provider '(Unknown Network)' %}</div>
-                <div class="small">
-                    Last Updated: January 1, 1970
+            <div class="dashboard__section-content">
+                <div class="dashboard-health-network__info-box">
+                    <div class="h3 dashboard-health-network__network-name">{% firstof request.user.userdetails.health_network.name request.user.userdetails.custom_provider '(Unknown Network)' %}</div>
+                    <div>
+                        Last Updated: January 1, 1970
```

Comment:
On the "Last Updated" text and elsewhere, there seems to be a medium gray in the prototype that isn't in the palette. Should we query Shane about these?